.github/CODEOWNERS

```diff
@@ -4,3 +4,8 @@
 # review when someone opens a pull request.
 
 *       @department-of-veterans-affairs/frontend-review-group
+
+# VSA
+src/applications/proxy-rewrite/redirects/crossDomainRedirects.json @department-of-veterans-affairs/vsa-frontend
```

Comment:
i pulled these from recent changes made by kelson and nick. this one is very specific but I didn't want to assume VSA wanted to take full ownership of the proxy-rewrite component